### PR TITLE
Use make install-all when installing citus if available

### PR DESCRIPTION
**Pull request by onurctirtir**
This is mainly to make valgrind testing happy
Tested with valgrind testing, works fine

### CODE REVIEW

**Comment by onurctirtir** [comment]
Tested with valgrind testing, works fine

**Comment by SaitTalhaNisanci** [comment]
What if we are installing an old version which doesn't have this install-all target?

**Comment by onurctirtir** [comment]
> What if we are installing an old version which doesn't have this install-all target?

maybe `make install-all || make install` ?

**Comment by SaitTalhaNisanci** [comment]
>maybe make install-all || make install ?

Makes sense.

File: fabfile/setup.py [hunk 1]
```diff
@@ -237,8 +236,13 @@ def build_enterprise():
...
         core_count = run('cat /proc/cpuinfo | grep "core id" | wc -l')
 
-        with hide('stdout'):
-            run('make -s -j{} install-all'.format(core_count))
+        install_citus(core_count)
+
+def install_citus(core_count):
+    with hide('stdout'):
+        # fall back to "make install" if "make install-all" is not available
+        run('make -s -j{} install-all || make -s -j{} install'.
```
Comment (SaitTalhaNisanci) [inline fabfile/setup.py:244]: Minor:
```python
make -s -j{core_count} install-all || make -s -j{core_count} install'.format(core_count=core_count)
```

**Review by SaitTalhaNisanci — APPROVED** [review]

File: fabfile/setup.py [hunk 2]
```diff
@@ -237,8 +236,13 @@ def build_enterprise():
...
 
-        with hide('stdout'):
-            run('make -s -j{} install'.format(core_count))
+        install_citus(core_count)
+
+def install_citus(core_count):
+    with hide('stdout'):
+        # fall back to "make install" if "make install-all" is not available
+        run('make -s -j{} install-all || make -s -j{} install'.
+            format(core_count, core_count))
```
Comment (onurctirtir) [inline fabfile/setup.py:245]:
```suggestion
        run('make -s -j{core_count} install-all || make -s -j{core_count} install'.\
            format(core_count=core_count))
```

**Pull request merged**